java/src/org/openqa/selenium/grid/config/Config.java

```diff
@@ -55,4 +59,19 @@ default <X> X getClass(String section, String option, Class<X> typeOfClass, Stri
       throw new IllegalArgumentException("Unable to find class: " + clazz, e);
     }
   }
+
+  default List<String> toEntryList(Map<String, Object> mapItem) {
```

Comment:
There's not base class for this sort of common functionality. What's the policy on implementing "utility" functions as default methods?

Reply:
Cant they be just utility methods from a package visible utility class?

Reply:
Sure, is there a standard naming convention for that sort of class?